### DISCUSSION TITLE
Перестали создаваться Заказы с сайта в МоемСкладе.

### DISCUSSION THREAD

**Discussion by Andrey20211** [wpcraft-ru/wooms#550] [General]
В статусе заказа на сайте пишет Заказ не передан в МойСклад.

![image](https://github.com/wpcraft-ru/wooms/assets/93029213/97bb7fc9-fb6b-42dc-86ca-718f42ba4ec8)

![image](https://github.com/wpcraft-ru/wooms/assets/93029213/df53be4b-dc7e-4c52-9efb-af7915cd1ca2)

**Comment by varikweru** [comment]
Посмотрите логи (Мой Склад - Журнал) - WooMS-Orders-дата
У меня такое бывало, когда не находился хотя бы один товар из заказа
